xml/ha_maintenance.xml

```diff
@@ -37,6 +37,50 @@
  </info>
  <sect1 xml:id="sec-ha-maint-shutdown-node">
   <title>Implications of Taking Down a Cluster Node</title>
+  <para>
+   In &sls; &hasi; 15 SP2, the way how cluster services were started and
+   stopped has been changed. &suse; recommends to use the &crmshell;. The
+   old commands with <command>systemctl</command> are still available, but
+   are recommended for experienced users only.
+   For details, refer to the &suse; blog article <link
+    xlink:href="https://www.suse.com/c/suse-high-availability-cluster-services-how-to-stop-start-or-view-the-status/"
+   />.
```

Comment:
Such link is good reference here to keep HA DOC simple and concise :)

All changes looks good to me

Thanks,
Roger

Reply:
Thanks Roger! :+1: